docs/runtime/transfer_learning.md

```diff
@@ -0,0 +1,199 @@
+# Transfer Learning
+
+Let's classify dogs and cats using the pre-trained model. It will be `mobilenet v2` in our case.
+
+## Preparing dataset
+The first step is conversion dataset (input data and expected outputs) to the binary format accepted by `ONE`. Note, that for many cases it can be achieved via [tf_dataset_converter](https://github.com/Samsung/ONE/tree/master/tools/generate_datafile/tf_dataset_converter).
+
+```python
+import os
+import tensorflow as tf
+import pathlib
+import numpy as np
+
+_URL = 'https://storage.googleapis.com/mledu-datasets/cats_and_dogs_filtered.zip'
+path_to_zip = tf.keras.utils.get_file('cats_and_dogs.zip', origin=_URL, extract=True)
+PATH = os.path.join(os.path.dirname(path_to_zip), 'cats_and_dogs_filtered')
+
+train_dir = os.path.join(PATH, 'train')
+validation_dir = os.path.join(PATH, 'validation')
+
+BATCH_SIZE = 32
+IMG_SIZE = (160, 160)
+
+def label_to_array(label):
+    arr = np.zeros(2, dtype=float)
+    arr[label] = 1.
+    tensor = tf.convert_to_tensor(arr, tf.float32)
+    return tensor
+
+train_dataset = tf.keras.preprocessing.image_dataset_from_directory(train_dir,
+                                                            shuffle=True,
+                                                            batch_size=BATCH_SIZE,
+                                                            image_size=IMG_SIZE)
+
+current_dir = pathlib.Path(__file__).parent
+with open(current_dir/'cats_and_dogs.input.bin', 'wb') as in_stream:
+    with open(current_dir/'cats_and_dogs.output.bin', 'wb') as out_stream:
+        for image,label in train_dataset:
+            # print(image.numpy().dtype)
+            in_stream.write(image.numpy().astype(np.float32).tobytes())
+            # print(label)
+            out_stream.write(label.numpy().astype(np.float32).tobytes())
+```
+
+## Feature extraction
+The first step to align the model to our problem can be inserting customs ops on the top of the model. The base of the model (with unchanged weights) remains the same.
+
+```python
+import tensorflow as tf
+import numpy as np
+
+BATCH_SIZE = 32
+IMG_SIZE = (160, 160)
+IMG_SHAPE = IMG_SIZE + (3,)
+
+base_model = tf.keras.applications.MobileNetV2(input_shape=IMG_SHAPE,
+                                               include_top=False,
+                                               weights='imagenet')
+
+base_model.trainable = False
+image_batch = tf.zeros([32, 160, 160, 3])
+feature_batch = base_model(image_batch)
+
+# feature extraction
+global_average_layer = tf.keras.layers.GlobalAveragePooling2D()
+feature_batch_average = global_average_layer(feature_batch)
```

Comment:
How about removing this line?
Since `feature_batch_average` is not used in this script, So when I first read the script, I wondered why this line was included. 

```suggestion
```

Reply:
ohh right, it's some leftover, removed ;)